node/sc/api_bridge.go

```diff
@@ -398,13 +406,6 @@ func (sb *SubBridgeAPI) DeregisterToken(cBridgeAddr, pBridgeAddr, cTokenAddr, pT
 		return ErrNoBridgeInfo
 	}
 
-	pTokenAddrCheck := cBi.GetCounterPartToken(cTokenAddr)
-	cTokenAddrCheck := pBi.GetCounterPartToken(pTokenAddr)
-
-	if pTokenAddr != pTokenAddrCheck || cTokenAddr != cTokenAddrCheck {
-		return errors.New("invalid toke pair")
-	}
-
```

Comment:
I think removing this code is unnecessary. Can you explain why you removed this code?

Reply:
The function `GetCounterPartToken()` lookup the `counterpartToken` property in `bridgeInfo` struct. The `counterpartToken` member is initialized when the function `RegisterToken()` is called. The `RegisterToken()` stands for recording the counterpart token address. If a node is rebooted, and no `RegisterToken()` is not called, `counterpartToken` becomes empty. Thus, the checker above is invalid.